tasks.py

```diff
@@ -115,8 +115,8 @@ def tutorials(c):
 @task
 def lint(c):
     check_dependencies(c)
-    c.run('flake8 copulas')
-    c.run('flake8 tests --ignore=D,SFS2')
+    c.run('flake8 copulas --max-expression-complexity=8')
```

Comment:
The standard limit is 7, but around 10 expressions have complexity of 7.5. Since it seemed silly to decrease the complexity of the expressions (they are just implementations of equations) I increased the cap.

Reply:
Is there a way to put this in a config instead of in the command?

Reply:
@fealho can you point at some of these expressions?
In general the complexity can be decreased by capturing some things in intermediate variables, so I suspect we would want to do that.

Reply:
Sure, I moved the limit back to 7 and I added the intermediate variables (you can check the specific examples in the code), I just think it doesn't necessarily improve code readability, so it's a silly change.